test_scripts/API/AddCommand/SDL_resends_WARNINGS_to_MOB_when_HMI_respond_WARNINGS_to_one_HMI_portions.txt

```diff
@@ -1,26 +1,27 @@
 ./test_scripts/API/AlertManeuver/ATF_AlertManeuver_Navi_AlertManeuver_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_Navi_AlertManeuver_TTS_Speak_SUCCESS
 ./test_scripts/API/AlertManeuver/ATF_AlertManeuver_TTS_Speak_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_Speak_Navi_AlertManeuve_SUCCESS
-./test_scripts/API/AlertManeuver/ATF_AlertManeuver_UNSUP_RES_TTS_Speak_WARNINGS_Navi_AlertMan.lua Checks_that_SDL_sends_UNSUPPORTED_RESOURCE_(success_true)_to_mobile_app_in_case_HMI_respond_UNSUP_RES_to_TTS_Speak_and_WARNINGS_to_Navi_AlertManeuver
 ./test_scripts/API/PerformInteraction/ATF_PerformInteraction_UI_PerformInteraction_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_PerformInteraction_VR_PerformInteraction_SUCCESS
 ./test_scripts/API/PerformInteraction/ATF_PerformInteraction_VR_PerformInteraction_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_VR_PerformInteraction_UI_PerformInteraction_SUCCESS
-./test_scripts/API/PerformInteraction/ATF_PerformInteraction_UNSUP_RES_to_VR_PerformInteract_WARNINGS_UI_PerformInteract.lua Checks_that_SDL_sends_UNSUPPORTED_RESOURCE_(success_true)_to_mobile_app_in_case_HMI_respond_UNSUPPORTED_RESOURCE_to_VR_PerformInteraction_and_WARNINGS_to_UI_PerformInteraction
-./test_scripts/API/PerformInteraction/ATF_PerformInteraction_UNSUP_RES_UI_PerfInterac_WARNINGS_VR_PerfInterac.lua Checks_that_SDL_sends_UNSUPPORTED_RESOURCE_(success_true)_to_mobile_app_in_case_HMI_respond_UNSUPPORTED_RESOURCE_to_UI_PerformInteraction_and_WARNINGS_to_VR_PerformInteraction
 ./test_scripts/API/PerformAudioPathThru/ATF_PerformAudioPathThru_UI_PerformAudioPathThru_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_UI_PerformAudioPathThru_TTS_Speak_SUCCESS
 ./test_scripts/API/PerformAudioPathThru/ATF_PerformAudioPathThru_TTS_Speak_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_Speak_UI_PerformAudioPathThru_SUCCESS
-./test_scripts/API/PerformAudioPathThru/ATF_PerformAudioPathThru_UNSUP_RES_UI_PerfAudioPathThru_WARNINGS_TTS_Speak.lua Checks_that_SDL_sends_UNSUPPORTED_RESOURCE_(success_true)_to_mobile_app_in_case_HMI_respond_UNSUPPORTED_RESOURCE_to_UI_PerformAudioPathThru_and_WARNINGS_to_TTS_Speak
\ No newline at end of file
+./test_scripts/API/AlertManeuver/ATF_TTS_IsReady_false_AlertManeuver_Navi_AlMan_WARNINGS.lua Precondition_TTS_IsReady_false_AlertManeuver_Navi_AlertManeuver_gets_WARNINGS_from_HMI_TTS_Speak_is_not_sent_to_HMI_and_SDL_resends_to_Mobile_UNSUPPORTED_RESOURCE_success_true
```

Comment:
@vvvakulenko, I think that you are mixing two cases for TTS.IsReady.
Case 1:
[TTS Interface] HMI does NOT respond to IsReady and mobile app sends RPC that must be splitted
Steps:
mob->SDL: RPC AlertManeuver
Expexted: 
SDL->HMI: TTS.Speak - should be send
Send HMI->SDL: TTS.Speak('UNSUPPORTED_RESOURCE')
SDL->HMI: Navi.AlertManeuver
Send HMI->SDL:  Navi.AlertManeuver (any successfull resultCode )
Expected:
SDL->mob: AlertManeuver_response (UNSUPPORTED_RESOURCE, success:true)
Description should be updated.

Case 2:
"TTS_Speak_is_not_sent_to_HMI" will be in case TTS.IsReady(available = false).
See requirements: 
[TTS Interface] TTS.IsReady(false) -> HMI respond with successfull resultCode to splitted RPC
[TTS Interface] TTS.IsReady(false) -> HMI respond with errorCode to splitted RPC

Steps:
mob->SDL: RPC AlertManeuver
Expexted: 
TTS.Speak should NOT be send
SDL->HMI: Navi.AlertManeuver
Send HMI->SDL:  Navi.AlertManeuver (any successfull resultCode/error_code )
Expected:
SDL->mob: AlertManeuver_response (UNSUPPORTED_RESOURCE, success:true) (error_code/success:false)

Please clarify with BA, should be tested case IsReady(available = false) in scope of this CRQ. In case "yes", additional scripts should be added.

The clarification should be done for all Interfaces and applied to all scripts.